setup.cfg

```diff
@@ -18,5 +18,7 @@ add_select = D212,D404
 convention = pep257
 
 [tool:pytest]
+markers =
+    latest_version(version=None): mark a test as understanding the latest known version of the IATI Standard to be the specified value
```

Comment:
This needs rephrasing. Maybe changing the focus from what it's doing to why it's doing it?